### DISCUSSION TITLE
Details for browsers

### DISCUSSION THREAD

**Discussion by piothPaproKee** [plausible/analytics#4356] [Anything Else]
Any chance to have details button for the `devices` section? Or is there an option to see all the data in the FE? 
![image](https://github.com/user-attachments/assets/2def00c0-4cad-42c2-9e62-5c1cd0a9f33a)

Thanks.

**Comment by ruslandoga** [comment]
👋 @piothPaproKee 

Could you please suggest that feature on https://feedback.plausible.io?

**Comment by metmarkosaric** [comment]
There’s now the Details view in the Devices report which allows you to see the full list of browsers, browser versions, OS and OS versions.
